README.md

```diff
@@ -11,8 +13,10 @@
 | [@cennznet/util](packages/util/README.md)     | cennznet specific utility functions                         |
 | [@cennznet/types](packages/types/README.md)   | cennznet specific type definitions                                |
 
-# How to contribute
+# Cennznet runtime module libraries 
```

Comment:
do we want this?

Reply:
We can have this section here, doesn't hurt.